src/app/project/cac/become-a-member.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, Input, OnInit } from '@angular/core';
+// import { HttpEventType } from '@angular/common/http';
```

Comment:
Is something planned for this import? If not, please remove

Reply:
Removed.